pages/recommended/dependency_management.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /recommendations/dependency-management/
+layout: default
+title: M**V Frameworks
+parent: Recommendations
+---
+
+The word "dependency" refers to all of the frameworks, libraries and other tools that your project relies on. *Dependency management* is the process by which tools are incorporated into your project, removed and updated (for instance, when you need a new version of [jQuery]). Here are the tools that we recommend for managing dependencies:
+
+## Bower
+#### Do not use Bower.
+
+It's not needed and should be phased out and replaced by npm. More information can be found here: http://gofore.com/ohjelmistokehitys/stop-using-bower/
```

Comment:
FWIW, our [accessibility guidelines](https://pages.18f.gov/accessibility/links/) suggest that the anchor text be more descriptive, such as `for more information, read the article [Why We Should Stop Using Bower – And How to Do It](http://gofore.com/ohjelmistokehitys/stop-using-bower/)`